pySDC/projects/PinTSimE/paper_PSCC2024/README.md

```diff
@@ -0,0 +1,40 @@
+# Spectral Deferred Corrections with Discontinuity Handling for Dynamic Power System Simulation
+
+The Python file `paper_plots.py` creates all the plots contained in the publication:
+
+**Title:** Spectral Deferred Corrections with Discontinuity Handling for Dynamic Power System Simulation
+
+**Authors:** Junjie Zhang, Lisa Wimmer, Robert Speck, Matthias Bolten, Kyrill Ho, and Andrea Benigni
+
+**Conference:** [![PSCC 2024](https://pscc2024.fr/)](https://pscc2024.fr/)
+
+Current status of the submission: ***submitted***
+
+## Plots for the discontinuous test DAE
+In order to reproduce the plots for the discontinuous test DAE, the following setting is used: The test DAE `DiscontinuousTestDAE`
+is simulated over the time domain with `t0=3.0` and `Tend=5.4` for different step sizes `dt_list = [1 / (2 ** m) for m in range(2, 9)]`. The fully implicit SDC-DAE sweeper `fully_implicit_DAE` solves the problem for different number of collocation
+nodes `nnodes=[2, 3, 4, 5]` at Radau IIa nodes `quad_type='RADAU-RIGHT` with LU preconditioner `QI='LU'` using `tol_hybr=1e-6` to solve the nonlinear system.
+SDC terminates either the maximum number of iterations `maxiter=45` or the residual tolerance `restol=1e-13` is satisfied.
+For event detection, if an event is found the step sizes will be adapted using the factor `alpha=0.95`. A founded event should satisfy the tolerance `epsilon_SE=1e-10`. 
```

Comment:
Yes, I mean that. You're completely right! I observed that the event is skipped after some time, because after some restarts it is possible that there is no sign change anymore, i.e., in the current step you has only values with one sign, then the solver goes ahead to the next step where the values of the state function has another sign.. pretty bad, and I assume the interpolation is at fault. One reason why I started to write a very meaningful test for `SwitchEstimator` at the TIME-X test hackathon. I guess this is a bad bug, and I want to find it to remove it.